src/pages/Offer/sections/InsuredAmount.tsx

```diff
@@ -80,12 +57,54 @@ export const InsuredAmount: React.SFC = () => (
             </TranslationsConsumer>
           </HeaderWrapper>
           <Table>
-            {AMOUNTS.map((amount) => (
-              <Row key={amount.key}>
-                <Col>{amount.title}</Col>
-                <Col>{amount.amount}</Col>
-              </Row>
-            ))}
+            <Row>
+              <Col>
+                <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_COL_ONE_TITLE">
+                  {(title) => title}
+                </TranslationsConsumer>
+              </Col>
+              <Col>
+                <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_COL_ONE_AMOUNT">
+                  {(amount) => amount}
+                </TranslationsConsumer>
+              </Col>
+            </Row>
+            <Row>
+              <Col>
+                <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_COL_TWO_TITLE">
+                  {(title) => title}
+                </TranslationsConsumer>
+              </Col>
+              <Col>
+                <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_COL_TWO_AMOUNT">
+                  {(amount) => amount}
+                </TranslationsConsumer>
+              </Col>
+            </Row>
+            <Row>
+              <Col>
+                <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_COL_THREE_TITLE">
+                  {(title) => title}
+                </TranslationsConsumer>
+              </Col>
+              <Col>
+                <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_COL_THREE_AMOUNT">
+                  {(amount) => amount}
+                </TranslationsConsumer>
+              </Col>
+            </Row>
+            <Row>
+              <Col>
+                <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_COL_FOUR_TITLE">
+                  {(title) => title}
+                </TranslationsConsumer>
+              </Col>
+              <Col>
+                <TranslationsConsumer textKey="OFFER_INSURED_AMOUNT_COL_FOUR_AMOUNT">
+                  {(amount) => amount}
+                </TranslationsConsumer>
+              </Col>
+            </Row>
```

Comment:
Maybe still use a map for a text keys arr for readability in code and less repeating?

Reply:
I'm inclined to agree. @palmfjord thoughts?

Reply:
yes, i agree 😬

Reply:
nice:) notice the section with placeholder images (Det här är hedvig) has been changed in master to the real ones.